app/src/test/java/org/simple/clinic/scanid/ShortCodeInputTest.kt

```diff
@@ -56,4 +56,18 @@ class ShortCodeInputTest {
     assertThat(result)
         .isEqualTo(Empty)
   }
+
+  @Test
+  fun `when entered code length is equal to 14 then validation should succeed`() {
```

Comment:
Can we also verify the error condition when the user enters a code that is more than 14 characters.